File: Logging/app/src/main/java/com/example/logging/MainActivity.java
package com.example.logging;

import android.content.Context;
import android.content.Intent;
import android.hardware.Sensor;
import android.hardware.SensorEvent;
import android.hardware.SensorManager;
import android.os.Bundle;
import android.support.wearable.activity.WearableActivity;
import android.util.Log;
import android.view.View;

import androidx.fragment.app.FragmentActivity;

import java.util.List;

public class MainActivity extends FragmentActivity {
    private static final String TAG = "MainActivity";



    public void onCreate(Bundle savedInstanceState) {
        Log.d(TAG, "onCreate()");
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);



        // Enables Always-on
//        setAmbientEnabled();
    }

    public final void pressButton(View view){
        Intent myIntent = new Intent(MainActivity.this, LoggingSensor.class);
        myIntent.putExtra("hi", 1);
        this.startActivity(myIntent);
    }
}

File: Logging/app/src/main/java/com/example/logging/MainActivity.kt
package com.example.logging

import android.os.Bundle
import android.support.wearable.activity.WearableActivity
import androidx.core.content.ContextCompat.getSystemService
import android.icu.lang.UCharacter.GraphemeClusterBreak.T



class MainActivityOLD : WearableActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // Enables Always-on
        setAmbientEnabled()

    }
}
